.gitignore

```diff
@@ -25,7 +25,6 @@ wheels/
 *.egg
 MANIFEST
 node_modules/
-frontend/static/frontend/main.js
```

Comment:
Mmm. Makes sense for a `build from github` style deploy system. Maybe we can at least rename the files / put them in some kind of `build` directory to make it more obvious what's going on here

Reply:
I agree with Drew here, though maybe for another PR?

Reply:
Yeah I reckon